src/renderer/template.ts

```diff
@@ -28,10 +35,23 @@ export async function renderTemplate(filepath: string, context: TemplateContext)
  * @private
  * @param filepath to import
  */
-function importComponent(filepath: string, context: TemplateContext): Promise<React.ReactElement> {
-  return new Promise((resolve) => {
-    import(filepath).then(component => { resolve(component.default !== undefined ? component.default(context) : undefined) })
-  })
+function importComponent(filepath: string, context: TemplateContext): Promise<React.ReactElement | undefined> {
+  return new Promise((resolve, reject) => {
+    try {
+      // we should import component only in NodeJS
+      if (require === undefined) resolve(undefined);
+      // remove from cache imported file
+      delete require.cache[require.resolve(filepath)];
+
+      const component = require(filepath);
+      if (!component) resolve(undefined);
```

Comment:
what is this for? if `filepath` here `const component = require(filepath);` is undefined then error is thrown, so when is the case that `component` is undefined?

Reply:
Yeah, you're right. Existed file return at least empty object, non existed throw error 😄 Thanks!